### DISCUSSION TITLE
redoc integration

### DISCUSSION THREAD

**Discussion by gugahoa** [outline/outline#1738] [Ideas]
Hi all,
A friend shared getoutline.com with me and I got really interested seeing as there's a full featured API. One of the first things that came to mind is to have design docs, service documentations and API specs (via redoc) live side by side in the same page.
I looked it up, and there doesn't seem to be some kind of redoc support in outline (for something like a redoc embedded page)

Is a feature like this desirable? And if so, could someone point me in the right direction so I can write a proof-of-concept?

**Comment by tommoor** [comment]
Hi @gugahoa – we do have [embeds for other services](https://www.getoutline.com/integrations) as I'm sure you've seen, but they usually take the form of a small interactive element.

We're open to integrations with any service and also to new forms of integration, but I don't think it would be desirable to embed an entire page from another product such as Redoc as it would be potentially quite confusing UX.

**Reply by matbrgz** [reply]
this list is missing some integrations.
